conans/model/build_info.py

```diff
@@ -441,6 +491,17 @@ def _aggregated_paths(self, item):
         setattr(self, "_%s_paths" % item, paths)
         return paths
 
+    def _aggregated_dict_paths(self, item):
```

Comment:
Again, maybe a bit too early generic at this point. Maybe it only needs to be implemented as BuildModulesPath thing, not as a general any-dictionary of paths.